elasticdl/python/common/args.py

```diff
@@ -200,6 +200,9 @@ def parse_worker_args(worker_args=None):
     parser.add_argument(
         "--worker_id", help="ID unique to the worker", type=int, required=True
     )
+    parser.add_argument(
+        "--num_workers", help="The number of workers", type=int, required=True
```

Comment:
This new required arg will break compatibility. Is this intended?

Reply:
Yes. If `num_workers=1`, we cannot use `CollectiveCommunicator` which cannot work if only a worker is configured. We can train the model locally without `CollectiveCommunicator` with a worker.

Reply:
Yes but you are adding a new required argument here in common args, which will potentially break existing usage.

Reply:
Workers are launched by the master. 
The master will always have`--num_workers` argument which will be passed to workers.

Thus, nothing will be broken.

Reply:
But need to add the argument in the unittest.

Reply:
Ok